tests/test_bronze_loading.py

```
@@ -21,191 +13,142 @@
-from src.models.enums import PlatformType, ProcessingStatusType
+from src.models.ingestion_batch import IngestionBatch
+from src.models.enums import PlatformType, IngestionBatchStatusType
 from src.utils.parquet_writer import read_parquet_to_schemas
 from src.schemas.parquet.app_review import AppReviewSchema
 
 
+def _make_crawler(test_db_session, temp_bronze_dir, enable_parquet=True):
+    """Helper: create AppStoreCrawler with mocked dependencies."""
+    from src.utils.parquet_writer import ParquetWriter
+
+    with patch.object(AppStoreCrawler, '__init__', lambda self, config_path: None):
```

Ruff (0.15.2)

[warning] Unused lambda argument: `self`

(ARG005)